uint8 team_number
uint8 team_colour
uint8 score
uint8 penalty_shot
uint16 single_shots
alice_msgs/RobotInfo[] players
